src/configuration.development.json

```diff
@@ -0,0 +1,54 @@
+{
+  "teamIds": [],
+  "groups": [],
+  "environment": "local",
```

Comment:
If the environment is "local", I'd recommend the file be named "configuration.local.json". Actually, I think using dashes is better than periods within the file name, so "configuration-local.json". I think "local" is more clear than "development".

Reply:
Yeah, that makes more sense. I'll change it.